app/src/main/java/edu/uc/forbesne/shoppingsidekick/ui/main/MainViewModel.kt

```diff
@@ -35,8 +35,7 @@ class MainViewModel : ViewModel() {
     private var _markets: MutableLiveData<ArrayList<Market>> = MutableLiveData<ArrayList<Market>>()
     var markets: MutableLiveData<ArrayList<Market>>? = null
         get() {
-            updateMarketsTotals()
-            return _markets
+            return updateMarketsTotals()
```

Comment:
This edit saves you a line of code, when using updateMarketsTotals() fun

Reply:
But it added two lines of code to the updateMarketTotals() function.  I'd leave this as it were.